stdlib/float.mli

```diff
@@ -81,6 +95,19 @@ val epsilon : float
 (** The difference between [1.0] and the smallest exactly representable
     floating-point number greater than [1.0]. *)
 
+val is_finite : float -> bool
+(** [is_finite x] says whether the number is finite i.e., not infinite
+   and not [nan]. *)
+
+val is_infinite : float -> bool
+(** [is_infinite x] says whether [x] is [infinity] or [neg_infinity]. *)
+
+val is_nan : float -> bool
+(** [is_nan x] returns [true] if and only if [x] represents a [nan]. *)
```

Comment:
("[nan]" instead of "a [nan]"?)

Reply:
I'm not sure what is the better (given that there are several NaNs, even though we do not distinguish them here).

Reply:
I am not sure, but I think the documentation tends
to use "nan" rather than "a nan" in other places.

Reply:
I can change it if you wish.  Or maybe reformulate it as _[x] is [nan]_.

Reply:
I do not have a strong opinion on this question;
maybe @Octachron or @gasche will.

Reply:
I like `[x] is [nan]`: it has the advantage of being correct when expanding it to `[x] is not a number`; moreover the Pervasives documentation already uses this wording.

Reply:
Done.